r/R/csv.R

```diff
@@ -189,9 +189,10 @@ read_tsv_arrow <- function(file,
 `arrow::csv::ParseOptions` <- R6Class("arrow::csv::ParseOptions", inherit = `arrow::Object`)
 `arrow::csv::ConvertOptions` <- R6Class("arrow::csv::ConvertOptions", inherit = `arrow::Object`)
 
-#' read options for the csv reader
+#' Read options for the Arrow file readers
 #'
-#' @param block_size Block size we request from the IO layer; also determines the size of chunks when use_threads is `TRUE`
+#' @param use_threads Whether to use the global CPU thread pool
```

Comment:
should this be in the function declaration below?  I only see blocksize

Reply:
I tried to fix this

Reply:
Thanks. This was a similar issue, where `json_read_options` was documented with `csv_read_options`, and the other function had the argument: https://enpiar.com/arrow-r-site/reference/csv_read_options.html

They should be the same so Wes's fix improves that.